tests/python/unittest/test_micro_model_library_format.py

```diff
@@ -208,7 +208,12 @@ def @main(%a : Tensor[(1, 2), uint8], %b : Tensor[(1, 2), float32], %c : Tensor[
                 {
                     "constants_size_bytes": json_constants_size_bytes,
                     "device": 1,
+                    "inputs": {
+                        "a": 2,
+                        "b": 8,
+                    },
```

Comment:
For future proofing, it'd be good to have more than just the size? But also dtype here?

```suggestion
                    "inputs": {
                        "a": {"dtype": "uint8", "size_in_bytes": 2},
                        "b": {"dtype": "float32", "size_in_bytes": 8},
                    },
```

Reply:
Added the check for the dtype